src/Examples.ts

```diff
@@ -0,0 +1,249 @@
+/**
+ * Inspired by create-next-app
+ */
+import chalk from 'chalk';
+import fs from 'fs';
+import got from 'got';
+import path from 'path';
+import prompts from 'prompts';
+import { Stream } from 'stream';
+import tar from 'tar';
+import { promisify } from 'util';
+
+// @ts-ignore
+const pipeline = promisify(Stream.pipeline);
+
+type RepoInfo = {
+  username: string;
+  name: string;
+  branch: string;
+  filePath: string;
+};
+
+export async function promptAsync(): Promise<string | null> {
+  const { value } = await prompts({
+    type: 'select',
+    name: 'value',
+    message: 'How would you like to start',
+    choices: [
+      { title: 'Default new app', value: 'default' },
+      { title: 'Examples from Expo', value: 'example' },
```

Comment:
```suggestion
      { title: 'Template from `${terminalLink('expo/examples', 'https://github.com/expo/examples')}`, value: 'example' },
```

maybe something like this so people can easily find the source of them and open prs etc?

what about something like this?

Reply:
(could also leave it as "Examples" but wanted to avoid saying the word twice in one sentence)

Reply:
I don't really wanna add an extra library to support linking to the location. https://packagephobia.now.sh/result?p=terminal-link